state/subnets.go

```diff
@@ -322,8 +322,27 @@ func (s *Subnet) updateSpaceName(spaceName string) (bool, error) {
 	return spaceNameChange && spaceName != "" && s.doc.FanLocalUnderlay == "", nil
 }
 
-// NetworkSubnet maps the subnet fields into a network.SubnetInfo.
-func (s *Subnet) NetworkSubnet() network.SubnetInfo {
+// AllSubnetInfos returns SubnetInfos for all subnets in the model.
+func (st *State) AllSubnetInfos() (network.SubnetInfos, error) {
+	subs, err := st.AllSubnets()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	result := make(network.SubnetInfos, len(subs))
+	for i, sub := range subs {
+		result[i] = sub.networkSubnet()
+	}
+	return result, nil
+}
+
+// networkSubnet maps the subnet fields into a network.SubnetInfo.
+// Note that this method should not be exported.
+// It is only called for on subnets are guaranteed, if Fan overlays,
```

Comment:
Is there a word missing here: "It is only called for on subnets are guaranteed,"

Reply:
Yep. Will fix.